packages/providers/fal/avatar-generation/fal_avatar/models/kling.py

```
@@ -0,0 +1,466 @@
+            prompt: Video generation prompt (use @Element1, @Element2, etc.)
+            reference_images: List of reference image URLs (max 4)
+            duration: Output duration ("5" or "10" seconds)
+            aspect_ratio: Video aspect ratio (16:9, 9:16, 1:1)
+            audio_url: Optional audio URL
+            face_id: Optional face ID from identify_face API
+
+        Returns:
+            Dict of validated parameters
+        """
+        if not prompt:
+            raise ValueError("prompt is required")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)